.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 .idea/
 .gradle/
+gradle/
```

Comment:
Is the top `.gradle` not redundant?

Reply:
actually gradle/ itself is redundant :) will remove